docs/kratos/self-service/hooks.mdx

```diff
@@ -137,7 +137,31 @@ after). For the `after` execution path of all flows, it binds the `identity`
 object into the jsonnet template as well. These objects are available through a
 `ctx` object. E.g. to create a body looking like `{ user_id: <some-id> }` to be
 send to the web hook end point, the jsonnet template would look like this:
-`function(ctx) { user_id: ctx.identity.id }`
+
+```jsonnet
+function(ctx) { user_id: ctx.identity.id }
+```
+
+#### Canceling Web Hooks
```

Comment:
```suggestion
#### Canceling webhooks
```

Reply:
Thank you, it is `webhook`. Regarding capitalization, we use new york times-style capitalization as a standard across all docs (cc @vinckr correct me if i am wrong):

```suggestion
#### Canceling Webhooks
```